frontend/src/e2e-test/generated/api-types.ts

```diff
@@ -189,7 +189,7 @@ export interface DevAssistanceFactor {
   childId: UUID
   id: UUID
   modified: HelsinkiDateTime
-  modifiedBy: UUID
+  modifiedByName: UUID
```

Comment:
These are still using the old type
```suggestion
  modifiedByName: string
```